generic/src/split.rs

```diff
@@ -3,6 +3,8 @@ use std::collections::BTreeSet;
 use prjunnamed_netlist::{Cell, Const, Design, FlipFlop, Net, Value};
 
 pub fn split(design: &mut Design) -> bool {
+    let _span = tracing::debug_span!("split").entered();
+
```

Comment:
Do we want a macro that makes it less wordy to replicate `#[tracing::instrument]` that we're avoiding due to deny(dtolnay)?

Reply:
Maybe, but I'm not sure. Getting the current function name is [annoying](https://stackoverflow.com/questions/38088067/equivalent-of-func-or-function-in-rust) and not that hard to do (although annoying to copy/paste); I guess we also have a lot of macros and I'm not sure if adding more for this is helping (at least the `tracing` crate is documented).

Reply:
This seems to work: https://github.com/prjunnamed/prjunnamed/pull/79/commits/c2baa4e9fc4b603513d748698c40008ea3428226#diff-e8fb311939422195b690f767be3dee583cbe00a5b192b15029158ddd21a63dedR63